src/protocol/mod.rs

```diff
@@ -283,11 +292,13 @@ impl Protocols {
                 (Ok(protocols), _) => {
                     let mut protocols = Protocols::from_array(protocols.expect_array()?)?;
                     protocols.add_unmocked_commands(object)?;
+                    protocols.add_interpreter(object)?;
```

Comment:
If we are given a `protocols:` object in the yaml file, here is where we add the interpreter. The function is defined above :)